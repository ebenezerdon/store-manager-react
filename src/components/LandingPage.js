import React from 'react'

const LandingPage = () => {
    return (
        <div>
            <h1> WELCOME TO LEARNGROUND </h1>
        </div>
    )
}

export default LandingPage
