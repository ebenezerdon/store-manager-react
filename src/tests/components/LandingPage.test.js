import React from 'react'
import { shallow } from 'enzyme'
import LandingPage from '../../components/LandingPage'

describe('LandingPage Component', () => {
    it('should render the Landing Page', () => {
        const component = shallow(<LandingPage />)
        expect(component.exists()).toBe(true)
        expect(component).toMatchSnapshot()
    })
})
