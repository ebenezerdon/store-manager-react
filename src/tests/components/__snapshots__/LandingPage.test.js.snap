// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingPage Component should render the Landing Page 1`] = `
<div>
  <h1>
     WELCOME TO LEARNGROUND 
  </h1>
</div>
`;
